src/test/java/org/broadinstitute/consent/http/db/UserPropertyDAOTest.java

```diff
@@ -0,0 +1,63 @@
+package org.broadinstitute.consent.http.db;
+
+import org.broadinstitute.consent.http.enumeration.UserFields;
+import org.broadinstitute.consent.http.enumeration.UserRoles;
+import org.broadinstitute.consent.http.models.User;
+import org.broadinstitute.consent.http.models.UserProperty;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.List;
+
+public class UserPropertyDAOTest extends DAOTestHelper {
+
+    @Test
+    public void testFindResearcherProperties() {
+        User user = createUserWithRole(UserRoles.RESEARCHER.getRoleId());
+
+        UserProperty suggestedInstitution = new UserProperty();
+        suggestedInstitution.setPropertyKey(UserFields.SUGGESTED_INSTITUTION.getValue());
+        suggestedInstitution.setPropertyValue("asdfasdf");
```

Comment:
Not a requirement, but consider using `RandomStringUtils` for this kind of task.